bundle/jsky.elevation.plot/src/main/java/jsky/plot/TargetDesc.java

```diff
@@ -5,11 +5,15 @@
  * $Id: TargetDesc.java 42349 2012-03-01 13:03:51Z swalker $
  */
 
-package jsky.coords;
+package jsky.plot;
 
 
 import edu.gemini.shared.util.immutable.Option;
+import edu.gemini.skycalc.Interval;
+import edu.gemini.skycalc.Union;
+import jsky.coords.WorldCoords;
```

Comment:
I've been trying to avoid `WorldCoords`. could/should we use the new `Coordinates` instead?